sr_data_visualization/src/sr_data_visualization/sr_data_visualizer_gui.py

```diff
@@ -92,10 +94,29 @@ def fill_layout(self):
             self.layout.addWidget(QLabel("No hand connected or ROS bag is not playing"), alignment=Qt.AlignCenter)
             return
 
+        self.hand_labels = []
+        self.hand_id = 0
+
+        if self.joint_prefix == "rh_":
+            self.hand_labels.append("rh")
+            self.hand_labels.append("lh")
+        else:
+            self.hand_labels.append("lh")
+            self.hand_labels.append("rh")
+
+        self.hand_id_selection = QComboBox()
+        self.hand_id_selection.addItems(self.hand_labels)
```

Comment:
```suggestion
        self.hand_id_selection = QComboBox()
        self.hand_id_selection.addItems(self.joint_prefix)
```

Reply:
This is because the plugin should stop if no hand is detected (self.joint_prefix would be None). So if the previous part passes (hands detected), self.joint_prefix should have the elements from ['rh', 'lh']